README.md

```diff
@@ -43,13 +43,27 @@ These instructions give the most direct path to a working Swift
 development environment. Options for doing things differently are
 discussed below.
 
-
 ### System Requirements
 
 macOS, Ubuntu Linux LTS, and the latest Ubuntu Linux release are the current
 supported host development operating systems.
 
-For macOS, you need [Xcode 9.0 beta 4](https://developer.apple.com/xcode/downloads/).
+To build for macOS, you always need the latest version of Xcode, currently
+[Xcode 9.0 beta 4](https://developer.apple.com/xcode/downloads/). You will
```

Comment:
Is someone going to keep this up to date? Could also link to developer.apple.com/downloads, although I believe that page is walled off behind an Apple ID login.